paddle/fluid/platform/cpu_info.cc

```diff
@@ -37,6 +37,11 @@ DEFINE_double(fraction_of_cpu_memory_to_use, 1,
               "reserve the rest for page tables, etc");
 DEFINE_uint64(initial_cpu_memory_in_mb, 500ul,
               "Initial CPU memory for PaddlePaddle, in MD unit.");
+DEFINE_uint64(cpu_max_chunk_size_in_mb, 4096ul,
```

Comment:
The title is for GPU, why add `cpu_max_chunk_size_in_mb` here?

Reply:
We are going to delay CPU implementation. I deleted here